ush/python/pygfs/task/marine_analysis.py

```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/env python3
+
+import copy
+import os
+from logging import getLogger
+import pygfs.utils.marine_da_utils as mdau
+import glob
+import re
+import netCDF4
+from multiprocessing import Process
+import subprocess
+import yaml
+from jcb import render
+
+from wxflow import (AttrDict,
+                    FileHandler,
+                    add_to_datetime, to_timedelta, to_YMD,
+                    parse_j2yaml,
+                    logit,
+                    Executable,
+                    Task,
+                    Template, TemplateConstants, YAMLFile)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+def parse_obs_list_file(obs_list_yaml_path):
+    # Get the list of observation types from the obs_list.yaml
+    obs_types = []
+    with open(obs_list_yaml_path, 'r') as file:
+        for line in file:
+            # Remove leading/trailing whitespace and check if the line is uncommented
+            line = line.strip()
+            if line.startswith('- !INC') and not line.startswith('#'):
+                # Extract the type using regex
+                match = re.search(r'\$\{MARINE_OBS_YAML_DIR\}/(.+)\.yaml', line)
+                if match:
+                    obs_types.append(str(match.group(1)))
+    return obs_types
```

Comment:
Not sure why we are still using `!INC` in the yamls.  They have all been replaced with jinja templates and includes.

Reply:
The "obs prep" script still depends on the old templating. We'll make the changes when we bring obs prep to this repo. 
I'll create an issue so we don't "forget".

Reply:
see [GDASApp/issues/1285](https://github.com/NOAA-EMC/GDASApp/issues/1285)